### PR TITLE
docs: Add process for "How to create a repo" 

### LINKED ISSUE #181: docs: Add process for "how to create a repository"
### Description

Hi @Pradumnasaraf, in the [how-to](https://oswp.study/#/pages/How-to/README) section of the website, if we can add how to create a repo, then it'll be easy to just refer to this page in future guides or processes which include creating a repository.

### Screenshots

_No response_

### PR DESCRIPTION
**Pull request by prnvtripathi**
## 🛠️ Fixes Issue

closes #181 

## 👨‍💻 Changes proposed

1. Changed [how-to](https://oswp.study/#/pages/How-to/README) page of the site and added a new row in the table at number 11, which links to the page on how to create a repo
2. Added new file in guide folder, create-repo.md
3. Changed link in [process of using github-pages](https://oswp.study/#/pages/How-to/guide/github-pages) in step 1 linking the new *"create repo"* page.

## ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## 📄 Note to reviewers

Please tell if any new changes are suggested. :)

<a href="https://gitpod.io/#https://github.com/Pradumnasaraf/open-source-with-pradumna/pull/198"></a>

### CODE REVIEW

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey, thank you for opening the PR.

Here are a couple of changes.
- Add the column to the last of the file in the `How to` `RAEDME` and use this image
How to process img
- Center the images in the `create-repo.md` docs

File: pages/How-to/guide/create-repo.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+#### Step 2
+1. Give your repository a nice name :)
+
+2. A good description.
+
+3. Choose the repository visibility. Public or Private
+
+4. And never forget to click on the checkbox saying add a README file.
+
```
Comment (Pradumnasaraf) [inline pages/How-to/guide/create-repo.md:26]:
```suggestion
#### Step 2

- Give your repository a nice name and description. 
- Choose the repository visibility. Public or Private. Also, it's good to have a README on the project, so check the checkbox.

```

**Comment by prnvtripathi** [comment]
hey @Pradumnasaraf, please check the changes and suggest if something else is needed

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**